test/EFCore.MySql.IntegrationTests/README.md

```diff
@@ -4,23 +4,25 @@ Integration and Performance Tests
 **Configuring the Database**
 
 1. Configure your MySQL database by opening the `config.json.example` file, specifying the connection string and saving the changed file as `config.json`.
-2. Run the `scripts/rebuild.sh` script on Linux or the `scripts/rebuild.ps1` script on Windows to rebuild all migrations. Any time you make changes to database models, run the rebuild script again.
+2. Run the `scripts/rebuild.ps1` script on Linux* or Windows to rebuild all migrations. Any time you make changes to the database models, run the rebuild script again.
+
+PS* - To install powershell on Linux read [this](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-linux) .
 
 **Running Integration Tests**
 
-1. Ensure that you configured the database (see previous paragraph).
+1. Ensure that you configured the database correctly (see previous paragraph).
 2. Run `dotnet test`. This will execute all tests in the Tests/ directory.
```

Comment:
We should mark `Tests/` as code using `` ` `` ticks.

Reply:
Will fix this.